pkgs/misc/vim-plugins/vim-plugin-names

```diff
@@ -41,6 +41,7 @@ Chiel92/vim-autoformat
 chikatoike/concealedyank.vim
 chikatoike/sourcemap.vim
 chrisbra/CheckAttach
+chrisbra/NrrwRgn
```

Comment:
please place in correct location `sort -udf`

Reply:
Done.